### DISCUSSION TITLE
Is Azure Data Studio Extension version still available?

### DISCUSSION THREAD

**Discussion by SoccerNerd** [MarkMpn/Sql4Cds#297] [General]
I am using v7.1.0 of SQL 4 CDS in Azure Data Studio and received a warning message that v7.2.2 is available. However, at first look I am now only seeing the version for XrmToolBox and SSMS. Am I overlooking the obvious?

**Comment by MarkMpn** [comment]
Go to the release page at https://github.com/MarkMpn/Sql4Cds/releases/tag/v7.2.2 and download the azuredatastudio-sql4cds-7.2.2.vsix file
